vscode-lean4/test/suite/bootstrap/bootstrap.test.ts

```diff
@@ -22,34 +23,10 @@ suite('Lean4 Bootstrap Test Suite', () => {
         assert(info, 'No InfoProvider export');
 
         // give it a extra long timeout in case test machine is really slow.
+        logger.log('Wait for elan install of Lean nightly build...')
 		await waitForInfoviewHtml(info, '4.0.0-nightly-', 600);
 
-        // test goto definition to lean toolchain works
-        await waitForActiveEditor();
```

Comment:
Did you intend to remove all this?

Reply:
yes, I copied this from the simple test which is also testing goto definition, but I didn't need that in the bootstrap test.